src/useChat.ts

```diff
@@ -68,7 +75,7 @@ export const useChat = ({
       queryValue: query,
       rerank: true,
       rerankNumResults: 7,
-      rerankerId: 272725718,
+      rerankerId: rerankerId,
```

Comment:
Nit, this can be simply `rerankerId`. JS will infer that we're assigning a variable of this name to a field of the same name. This comment applies in a few other places.